docs/providers/aws/guide/serverless.yml.md

```diff
@@ -16,115 +16,272 @@ layout: Doc
 
 Here is a list of all available properties in `serverless.yml` when the provider is set to `aws`.
 
+## Root properties
+
 ```yml
 # serverless.yml
 
-service: myService
+# Service name
+service: myservice
 
+# Framework version constraint: '3', '>=2.33', '^2.33'
 frameworkVersion: '3'
-configValidationMode: warn # Modes for config validation. `error` throws an exception, `warn` logs error to console, `off` disables validation at all. The default is warn.
-enableLocalInstallationFallback: false # If set to 'true', guarantees that it's a locally (for service, in its node_modules) installed framework which processes the command
-useDotenv: false # If set to 'true', environment variables will be automatically loaded from .env files
-deprecationNotificationMode: warn:summary # 'warn' reports deprecations on the go, 'error' will result with an exception being thrown on first approached deprecation
 
-disabledDeprecations: # Disable deprecation logs by their codes. Default is empty.
+# Configuration validation: 'error' (fatal error), 'warn' (logged to the output) or 'off' (default: warn)
+# See https://www.serverless.com/framework/docs/configuration-validation
+configValidationMode: error
```

Comment:
Good point, yes this is intentional: since we now always document the default value, it makes less sense to duplicate the default value below.

On top of that, the main use case where you want to copy a line from the docs and paste it into your project is to override the default. E.g. on this line here the main use case is for users to opt-in the error mode. So it makes more sense to document a different value (when possible, the most popular alternative to the default).